protoc-artifacts/pom.xml

```diff
@@ -8,7 +8,7 @@
   </parent>
   <groupId>com.google.protobuf</groupId>
   <artifactId>protoc</artifactId>
-  <version>3.7.0</version>
+  <version>3.7.0-rc1</version>
```

Comment:
3.7.0-rc-1

Reply:
I also already pushed the rc1 protoc artifacts so this one is too late to change as well.